osmoutils/cli_helpers.go

```diff
@@ -64,16 +64,6 @@ func ParseSdkDecFromString(s string, separator string) ([]sdk.Dec, error) {
 	return parsedDec, nil
 }
 
-func ParseSdkValAddressFromString(s string, separator string) []sdk.ValAddress {
-	var parsedAddr []sdk.ValAddress
-	for _, addr := range strings.Split(s, separator) {
-		valAddr := sdk.ValAddress([]byte(addr))
-		parsedAddr = append(parsedAddr, valAddr)
-	}
-
-	return parsedAddr
-}
```

Comment:
Why was this removed?

Reply:
I found a simpler inline way of handling this [here](https://www.youtube.com/watch?v=zBHRsYhYb-o)

also since validateBasic() checks validator address validity so this was unnecessary